### PR TITLE
Proofread mutation testing

### PR DESCRIPTION
**Pull request by yangelova**
Those are some small beauty changes to the text. Overall it was really nice and easy to follow.

### CODE REVIEW

File: chapters/intelligent-testing/mutation-testing.md [hunk 1]
```diff
@@ -608,7 +608,7 @@ Finally, you can also see more detailed results in the source code and check whi
 
 
 **Exercise 1.**
-"*Crimes* happen in a *city*. One way for us to know that the *police* is actually able to detect these *crimes*, we can *simulate crimes* and see whether the *police* is able to detect them."
+"*Crimes* happen in a *city*. One way for us to know that the *police* is actually able to detect these *crimes*, is to *simulate crimes* and see whether the *police* is able to detect them."
```
Comment (mauricioaniche) [inline chapters/intelligent-testing/mutation-testing.md:611]: remove the comma, just after *crimes*?

File: chapters/intelligent-testing/mutation-testing.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ small errors, will also be able to detect larger, more complex errors.
 
 ## Terminology
 
-To talk about mutation testing in a depth way, let us define some terms:
+To talk about mutation testing in an in depth way, let us define some terms:
```
Comment (mauricioaniche) [inline chapters/intelligent-testing/mutation-testing.md:165]: Great catch!

Is it _in depth_ or _in-depth_?

File: chapters/intelligent-testing/mutation-testing.md (hunk 1 above)
Comment (yangelova) [reply chapters/intelligent-testing/mutation-testing.md]: Done ✅

File: chapters/intelligent-testing/mutation-testing.md (hunk 2 above)
Comment (yangelova) [reply chapters/intelligent-testing/mutation-testing.md]: In this case it is _in-depth_ indeed.

**Pull request merged**